geonode/settings.py

```diff
@@ -2029,3 +2029,16 @@ def get_geonode_catalogue_service():
 # -- END Settings for MONITORING plugin
 
 CATALOG_METADATA_TEMPLATE = os.getenv("CATALOG_METADATA_TEMPLATE", "catalogue/full_metadata.xml")
+UI_DEFAULT_MANDATORY_FIELDS = [
+    'id_resource-title', 
+    'id_resource-abstract', 
+    'id_resource-language',
+    'id_resource-license',
+    'id_resource-regions',
+    'category_form', 
```

Comment:
@mattiagiupponi ok for dates, but groups are only required in case the advanced workflow is activated. In theory it should be moved to Optional and be treated as a requried field only in case the advnaced workflow is on.

Let's keep it as it is for the moment.